src/amo/reducers/viewContext.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  VIEW_CONTEXT_EXPLORE,
+  VIEW_CONTEXT_HOMEPAGE,
+  VIEW_CONTEXT_SET,
+} from 'core/constants';
+
+
+export type ViewContextType = {|
+  context: ADDON_TYPE_EXTENSION | ADDON_TYPE_THEME | VIEW_CONTEXT_EXPLORE |
+    VIEW_CONTEXT_HOMEPAGE,
```

Comment:
Oddly it doesn't seem to be working though. They're the only context we allow and use but if I change the values or even change that union to "number" flow still says "No errors!". So it doesn't seem to be working.